cmd/limactl/disk.go

```diff
@@ -101,7 +101,8 @@ func diskCreateAction(cmd *cobra.Command, args []string) error {
 	}
 
 	if err := qemu.CreateDataDisk(diskDir, format, int(diskSize)); err != nil {
-		return fmt.Errorf("Failed to create %s disk in %q", format, diskDir)
+		_ = os.RemoveAll(diskDir)
```

Comment:
Maybe we should handle the returned error?

Like the following:
```go
rerr := os.RemoveAll(diskDir)
if rerr != nil {
	err = errors.Join(err, fmt.Errorf("remove all files: %w", rerr))
}
return fmt.Errorf("Failed to create %s disk in %q: %w", format, diskDir, err)
```

Reply:
Done